index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <div class="header">
+          <div class="bar1"> 9:41
+          <img src="/images/Group 4.png" alt="icons"></div>
+          <div class="bar2"> 
+            <h1>TRACY MIRANJA</h1>
+          <img src="/images/Icon - Menu.png" alt="togglenav">
```

Comment:
- I thin it will be best to wrap your hamburger menu in a `<nav>` tag because `nav` tag is for semantics reason, and user experience for mobile devices. 🌸 

You can do something like this 👇🏽 
```
<nav>
 <img src="./images/Icon - Menu.png" alt="togglenav">
<nav>
```
Can we fix this?? 🤔